lib/Inspection/Access.h

```diff
@@ -329,6 +329,16 @@ struct Access<VPackBuilder> : AccessBase<VPackBuilder> {
   }
 };
 
+template<typename T>
+struct Access<std::reference_wrapper<T>>
+    : AccessBase<std::reference_wrapper<T>> {
+  template<class Inspector>
+  static auto apply(Inspector& f, std::reference_wrapper<T>& x) {
+    static_assert(!Inspector::isLoading);
```

Comment:
Why does this only work for writing?

Reply:
I added a comment

Reply:
I think we should rather use the static_assert message instead of a comment:
`static_assert(!Inspector::isLoading, "...");`

Reply:
Oh yes, I like that.